bin/version-include.sh

```diff
@@ -63,50 +64,82 @@ function deployV4MInfo() {
   helm upgrade --install \
     -n "$NS" \
     --values $valuesYAML \
-    v4m ./v4m-chart
+    $releaseName ./v4m-chart
+
+  getHelmReleaseVersion "$NS" "$releaseName"
 }
 
 function removeV4MInfo() {
   NS=$1
+  releaseName=$2
   if [ -z "$NS" ]; then
     log_error "No namespace specified for removing Viya Monitoring for Kubernetes version information"
     return 1
   fi
   log_info "Removing Viya Monitoring for Kubernetes version information"
-  helm uninstall -n "$NS" v4m
+  helm uninstall -n "$NS" "$releaseName"
```

Comment:
We should confirm the release exists.  If we don't, this helm command will return a non-zero exit code...and errexit (set -e) will exit.

Reply:
Although it seemed to continue working despite producing an error, I have I added logic to check if the namespace/releaseName combination exists before attempting to do perform the uninstall.  While the code I implemented does work as I expect it to, it keeps giving me a message saying that there are too many square brackets that I'm trying to work through.

Reply:
It looks like the helm list command will return an empty set of square brackets if the specified release doesn't exist and you've requested yaml output.  
```
-bash-4.2$ helm list -n logging --filter "^does_not_exist$" -o yaml
[]
```
That might be part of what you are running into.